recipes/ribotin/meta.yaml

```diff
@@ -1,18 +1,18 @@
 {% set name = "ribotin" %}
-{% set version = "1.3" %}
+{% set version = "1.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/maickrau/{{ name }}/files/14571457/{{ name }}.tar.gz
-  sha256: 7eda750e4fd1064eedf3016f5a49d6b6994e336619fc3b5e4afe4655b2873b70
+  url: https://github.com/user-attachments/files/18780237/{{ name }}.tar.gz
```

Comment:
What is this url ?
https://github.com/user-attachments/files says that is an archived repository

Reply:
That's the URL that github gave the source code tarball when I added it as a file attachment to the release in the ribotin repository. Release is at https://github.com/maickrau/ribotin/releases/tag/v1.4. The `user-attachments/files` repository is not related to this file.

Reply:
Can you use https://github.com/maickrau/ribotin/archive/refs/tags/v1.4.tar.gz instead ?

Reply:
Unfortunately no because github source tarballs don't include submodules (see https://github.com/dear-github/dear-github/issues/214). The tarball in the release url was generated with [git-archive-all](https://github.com/Kentzo/git-archive-all) instead.

Reply:
I see!
In that case I'd recommend you to use `git_url:` instead. 
See an example at
https://github.com/bioconda/bioconda-recipes/blob/5a3e067d748db25276dc7ed3dda32c8b8794fad0/recipes/meryl/meta.yaml#L10-L11
and
https://github.com/bioconda/bioconda-recipes/blob/5a3e067d748db25276dc7ed3dda32c8b8794fad0/recipes/meryl/meta.yaml#L54-L56

Reply:
Nice, git_url seems to work fine